CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -657,7 +657,15 @@ public let ATTRIBUTE_NODES: [Node] = [
     children: [
       Child(
         name: "Label",
-        kind: .node(kind: .token),
+        kind: .token(choices: [
+          .keyword(text: "target"),
+          .keyword(text: "availability"),
+          .keyword(text: "exported"),
+          .keyword(text: "kind"),
+          .keyword(text: "spi"),
+          .keyword(text: "spiModule"),
+          .keyword(text: "available"),
+        ]),
```

Comment:
Will remove this @ahoppen  as the parser fails in some test cases. 
We are now allowed to provide `e` as a label. 

Will be added in #1727